### PR TITLE
[IMP] hall of fame according to VSC

### PR DESCRIPTION
**Pull request by VincentSchippefilt**
started the 2024 version

## Description:

description of this task, what is implemented and why it is implemented that way.

Task: [TASK_ID](https://www.odoo.com/odoo/2328/tasks/TASK_ID)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: hall-of-fame.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Remi (RAR): Freeze pane (with ANHE), Accounting formulas with (LUL), currency formats with LAA
+- Chenyun (CHYA): import/export images/text wrapping, generalized mixin of python spreadsheet (abstract spreadsheet), import/export in excel, multiple improvements in ranges, charts, sheets, edgescrolling
+- Mathieu (MAMU): images
+
+## (fin 2024)
+
+- Mehdi (MERA) and Florian (FLDA) join us
+- Alexis (LAA): vectorization of formulas, THE recompute zones
+- Lucas (LUL): rainbow errors detection, so many performance improvements, the calculated measures in pivot <3
+- Alexis (LAA), Antony (ANHE), Mehdi (MERA): POC about AI: use it to create pivots automatically based on the models/fields of Odoo --> AI is not ready for us
```
Comment (anhe-odoo) [inline hall-of-fame.md:31]: Anthony :'(

**Review by anhe-odoo — CHANGES_REQUESTED** [review]

File: hall-of-fame.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Vincent Schippefilt, the 10 Feb 2023 at 10:52 AM
+
+## (fin 2020)
+
+puisqu'on a des nouveaux, je met le "wall-of-fame according to VSC":
+
+- Pierre (pro) : 1er membre de l'équipe, à implémenté la plus grosse partie de =PIVOT, et les systèmes des registries tout entier
+- Lucas (lul) : à implémenté la dernière version du composer, des graph, le input selection et les 😠 de templates avec FLE
```
Comment (rrahir) [inline hall-of-fame.md:8]:
```suggestion
- Lucas (lul) : a implémenté la dernière version du composer, des graph, le input selection et les 😠 de templates avec FLE
```

think it's present in lots of places ^^

File: hall-of-fame.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- Florent (FLE) : à implémenté la plupart des manipulations de grilles et tout ce qui en découle dans le différents plugins, et les 🥲 de templates avec LUL
+- Alexis (LAA) : à implémenté toutes les formules (+/- 160 à ce jour)
+- Géry (GED) : à implémenter tout. GED est un dieu vivant et nous pauvre mortels ne pouvons qu'espérer arriver un jour à sa cheville. Maintenant il est retourné Guruté l'équipe framework JS que j'ai abandonné lâchement en créant l'équipe BI avec PRO
```
Comment (rrahir) [inline hall-of-fame.md:11]:
```suggestion
- Géry (GED) : a implémenté tout. GED est un dieu vivant et nous pauvre mortels ne pouvons qu'espérer arriver un jour à sa cheville. Maintenant il est retourné Guruter l'équipe framework JS que j'ai abandonné lâchement en créant l'équipe BI avec PRO
```

File: hall-of-fame.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+- Vincent (VSC): à implémenté la première version du composer, des graphs et du conditional formatting. Je suis aussi fier d'avoir trouvé l'algo de remplissage des pivot (implémenté par PRO) et le système de plugins (implémenté par GED), ainsi que les quelques refactorings qu'on a prévu de faire.
+- PRO: global filter, autofill pivots
```
Comment (rrahir) [inline hall-of-fame.md:13]: [this](https://en.wikipedia.org/wiki/Dissociative_identity_disorder) ?

source: I know people in the psychiatry field

File: hall-of-fame.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+- Lucas (LUL) : Multi-user avec PRO, sharing, Dashboard structures avec PRO (2022), refactoring master
+- Anthony (ANHE): handcrafted svg icons, formula arrays (with ADRM)
+- Remi (RAR): Freeze pane (with ANHE), Accounting formulas with (LUL), currency formats with LAA
+- Chenyun (CHYA): import/export images/text wrapping, generalized mixin of python spreadsheet (abstract spreadsheet), import/export in excel, multiple improvements in ranges, charts, sheets, edgescrolling
+- Mathieu (MAMU): images
+
+## (fin 2024)
+
+- Mehdi (MERA) and Florian (FLDA) join us
+- Alexis (LAA): vectorization of formulas, THE recompute zones
```
Comment (rrahir) [inline hall-of-fame.md:29]:
```suggestion
- Alexis (LAA): vectorization of formulas, new recompute zones algorithm
```

File: hall-of-fame.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- PRO: global filter, autofill pivots
+
+## (fin 2023)
+
+- Alexis (LAA), Anthony (ANHE) : evaluation multi-cell (2023), dynamic format evaluation (2022), text wrapping
+- Adrien (ADRM) : import excel, data filters, grouping, data validation, localisations (numbers/date formats)
+  financial formulas (> 40), multi-line (especially in composer)
+- Lucas (LUL) : Multi-user avec PRO, sharing, Dashboard structures avec PRO (2022), refactoring master
+- Anthony (ANHE): handcrafted svg icons, formula arrays (with ADRM)
+- Remi (RAR): Freeze pane (with ANHE), Accounting formulas with (LUL), currency formats with LAA
```
Comment (rrahir) [inline hall-of-fame.md:22]:
```suggestion
- Remi (RAR): Freeze pane (with ANHE), Accounting formulas (with LUL), currency formats (with LAA)
```

File: hall-of-fame.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+- Vincent (VSC): à implémenté la première version du composer, des graphs et du conditional formatting. Je suis aussi fier d'avoir trouvé l'algo de remplissage des pivot (implémenté par PRO) et le système de plugins (implémenté par GED), ainsi que les quelques refactorings qu'on a prévu de faire.
+- PRO: global filter, autofill pivots
+
+## (fin 2023)
+
+- Alexis (LAA), Anthony (ANHE) : evaluation multi-cell (2023), dynamic format evaluation (2022), text wrapping
+- Adrien (ADRM) : import excel, data filters, grouping, data validation, localisations (numbers/date formats)
+  financial formulas (> 40), multi-line (especially in composer)
+- Lucas (LUL) : Multi-user avec PRO, sharing, Dashboard structures avec PRO (2022), refactoring master
+- Anthony (ANHE): handcrafted svg icons, formula arrays (with ADRM)
```
Comment (pro-odoo) [inline hall-of-fame.md:21]: implementation of formula arrays system was with Alexis

Comment (anhe-odoo) [reply hall-of-fame.md:21]: Yes, but the implementation of the formula (Transpose, ...) was done with Adrien

**Comment by VincentSchippefilt** [comment]
robodoo r+

**Pull request closed without merging**